CMakeLists.txt

```diff
@@ -872,6 +872,14 @@ cmake_dependent_option(
   "USE_CUDA OR USE_ROCM;NOT MSVC"
   OFF)
 
+cmake_dependent_option(
+  USE_FBGEMM_GENAI
```

Comment:
could you also add this to https://github.com/pytorch/pytorch/blob/4d5d56a30edf3b524f907ae07b7985d6698a2c4b/setup.py